### PR TITLE
Docs: theme updates V

### PR DESCRIPTION
**Pull request by lunelson**
These updates sync with theme release 0.7.11

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "@mdx-js/mdx": "^1.6.16",
     "@mdx-js/react": "^1.6.16",
     "@philpl/buble": "^0.19.7",
-    "@rasahq/docusaurus-theme-tabula": "^0.7.8",
+    "@rasahq/docusaurus-theme-tabula": "^0.7.11",
```
Comment (m-vdb) [inline docs/package.json:45]: Did you have these changes reviewed by someone?

**Review by m-vdb — CHANGES_REQUESTED** [review]

File: docs/package.json (hunk 1 above)
Comment (lunelson) [reply docs/package.json:45]: No, I didn't :shrug

**Comment by lunelson** [comment]
@m-vdb similar changes to `changelog.mdx` source made in this, as in rasa-x

File: CHANGELOG.mdx [hunk 2]
```diff
@@ -17,32 +17,32 @@ https://github.com/RasaHQ/rasa/tree/master/changelog/ . -->
 <!-- TOWNCRIER -->
 
 ## [major.minor.patch] - 2020-09-16
-                                   
+
```
Comment (m-vdb) [inline CHANGELOG.mdx:20]: can you merge master in your branch? I think you'll have some conflicts as this section and several lines below were removed :)

**Review by m-vdb — APPROVED** [review]

**Pull request merged**